src/containers/pages/Home/Home.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { Col, Row } from 'antd';
+import './Home.css';
+
+const Home = (): JSX.Element => {
```

Comment:
I think the more generally accepted way to write this type definition is something like this:

```ts
const Component: React.FC<Props> = (props) => (
```
or where there are no props

```ts
const Component: React.FC = () => (
```